articles/cognitive-services/form-recognizer/concept-invoices.md

```diff
@@ -38,6 +38,11 @@ You will need an Azure subscription ([create one for free](https://azure.microso
 
 [!INCLUDE [input requirements](./includes/input-requirements-receipts.md)]
 
+## Supported locales 
+
+* **Pre-built Receipt v2.1-preview.3** (Public Preview) only supports invoices in the EN-US locale.
```

Comment:
en-US (2 instances)

Since 2.1 GA is not yet available, I would not add that line.  Once it is available, I would remove the preview line.

Reply:
Thank you @bojunehsu